packages/blade/src/components/Typography/Title/__tests__/__snapshots__/Title.web.test.tsx.snap

```diff
@@ -0,0 +1,46 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Title /> should render Title with variant "large" 1`] = `
+<div>
+  <h1
+    class="StyledBaseTextweb__StyledBaseText-sc-1t5p7u8-0 dJNfSh"
+    color="hsla(217, 18%, 45%, 1)"
+    font-family="Lato, -apple-system, BlinkMacSystemFont, \\"Segoe UI\\", Roboto, Helvetica, sans-serif, \\"Apple Color Emoji\\", \\"Segoe UI Emoji\\", \\"Segoe UI Symbol\\""
+    font-size="2.5rem"
+    font-style="normal"
+    font-weight="700"
+  >
+    Displaying Landing Page Title
+  </h1>
+</div>
+`;
+
+exports[`<Title /> should render Title with variant "medium" 1`] = `
+<div>
+  <h2
+    class="StyledBaseTextweb__StyledBaseText-sc-1t5p7u8-0 ieFrrL"
+    color="hsla(216, 16%, 60%, 1)"
+    font-family="Lato, -apple-system, BlinkMacSystemFont, \\"Segoe UI\\", Roboto, Helvetica, sans-serif, \\"Apple Color Emoji\\", \\"Segoe UI Emoji\\", \\"Segoe UI Symbol\\""
+    font-size="1.75rem"
+    font-style="normal"
+    font-weight="700"
+  >
+    Displaying Landing Page Title
+  </h2>
+</div>
+`;
+
+exports[`<Title /> should render Title with variant "small" 1`] = `
+<div>
+  <h3
+    class="StyledBaseTextweb__StyledBaseText-sc-1t5p7u8-0 jOhoak"
+    color="hsla(217, 56%, 17%, 1)"
+    font-family="Lato, -apple-system, BlinkMacSystemFont, \\"Segoe UI\\", Roboto, Helvetica, sans-serif, \\"Apple Color Emoji\\", \\"Segoe UI Emoji\\", \\"Segoe UI Symbol\\""
+    font-size="1.5625rem"
```

Comment:
Hmmm, I never noticed this. But these props should not be in the html. 🤔

Reply:
So styled components seem to injecting this, I wonder why. Only for handful of props it does, if the prop name is `fontSize`, 'fontWeight' etc. Won't do it for other props like `lineHeight`

Reply:
yep we need to filter these props using [`shouldForwardProp`](https://styled-components.com/docs/api#shouldforwardprop) but since it's not affecting anything I didn't wanted to add another layer to this.